src/Offer/ReadModel/JSONLD/NewPropertyPolyfillOfferRepository.php

```diff
@@ -78,4 +79,17 @@ function (array $subEvent) {
 
         return $json;
     }
+
+    private function polyfillEmbeddedPlaceStatus(array $json): array
+    {
+        if (!isset($json['location']['status']) || isset($json['location']['status']['type'])) {
```

Comment:
Shouldn't we set a default `Available` if there's no status? Or does the calendar summary not need it? I assumed it did because it was crashing on the incorrectly formatted status.

Reply:
The calendar summary doesn't care about the status of the location, but deserialization indeed crashes if the format is wrong

Reply:
We can always add it for completeness' sake though

Reply:
Makes sense!

Reply:
The default status is always set on the place and when the event gets projected the place is embedded with a status inside the event projection.
But the issue was temporary, only during the timeframe that status got projected without type.

Reply:
Does that mean I should revert the last commit?

Reply:
I don't think it is needed...

Reply:
I think we need it, for example for events who's place has not been explicitly updated.
See an example here: https://io.uitdatabank.be/event/03d993c3-cee4-4394-bca5-180fc5b05027
The event gets a status polyfilled, but the place doesn't because it's cached on the event JSON-LD and the place doesn't get a status polyfilled.